backend/app/api/webhooks.py

```
@@ -272,6 +272,7 @@ async def stripe_payment_intent_succeeded(
     tenant_info=Depends(tenant_secrets),
 ):
     raw_body = getattr(request.state, "original_body", None) or await request.body()
+    stripped_body = await request.body()
```

Ruff (0.15.0)

[error] Local variable `stripped_body` is assigned to but never used

Remove assignment to unused variable `stripped_body`

(F841)